lib/execution_engine2/sdk/EE2Status.py

```diff
@@ -474,7 +474,7 @@ def check_jobs(
                 del mongo_rec["_id"]
                 mongo_rec["retry_count"] = len(job["retry_ids"])
                 mongo_rec["job_id"] = str(job.id)
-                mongo_rec["parent_job_id"] = job.job_input.parent_job_id
+                mongo_rec["parent_job_id"] = job.job_input.parent_job_id if job_job.input else None
```

Comment:
How can `job_input` ever be `None`?

Reply:
Also looks like there's a switched _ and . in job_job.input

Reply:
For parent_jobs, there is no job input